caravel/bin/caravel

```diff
@@ -125,6 +125,35 @@ def refresh_druid():
             "[" + cluster.cluster_name + "]")
     session.commit()
 
+@manager.option(
+    '-p', '--prefix', default='data_',
```

Comment:
I think no filter is not good, because main database (for caravel system use) have many table not store data, It doesn't need sync.